specs/latest/1.0/index.html

```diff
@@ -3575,6 +3575,11 @@ <h2 class="no-toc">Table of contents</h2>
 
 </dl>
 
+<p>
+If the TexImageSource is an ImageBitmap, then these three parameters will be ignored. Instead
+the equivalent <a href="https://wiki.whatwg.org/wiki/ImageBitmap_Options">ImageBitmapOptions</a>
+should be used to create an ImageBitmap with desired format.
```

Comment:
To avoid confusion, the entry points taking the TexImageSource union type should be modified somehow to more clearly point out that these pixel storage parameters don't apply when an ImageBitmap is being uploaded. Re-reading the docs for texImage2D, I think they could use a larger restructuring, so I'll take care of that after merging this pull request.